oreClient/src/main/resources/assets/components/Pagination.vue

```diff
@@ -0,0 +1,84 @@
+<template>
```

Comment:
This breaks when on the second page and searching for something which only returns one page.